### PR TITLE
Configuration updates

### PR DESCRIPTION
**Pull request by maciaszczykm**
### What this PR does / why we need it
- Move .npmrc to package.json as we keep most of the config here anyways
- Remove myself and @floreks from the default assignees
- Bump Go and dependencies

### Release note
```release-note
NONE
```

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -18,9 +18,6 @@ updates:
     directory: "/"
     schedule:
       interval: daily
-    assignees:
```
Comment (Waseem826) [inline .github/dependabot.yml:21]: Instead of removing the `assignees` entry you can add me (`waseem826`) or any other frontend dev here.

**Review by Waseem826 — CHANGES_REQUESTED** [review]

**Comment by KhizerRehan** [comment]
Looks good! @Waseem826 once your queries are clarified approve and add label it to it.

**Comment by maciaszczykm** [comment]
@Waseem826 @KhizerRehan PTAL

**Review by Waseem826 — APPROVED** [review]

**Review by KhizerRehan — APPROVED** [review]

**Pull request merged**